### PR TITLE
vm-run: support adding multiple disks

### PR DESCRIPTION
**Pull request by jelly**
Allow -S to be repeated so one can easily test RAID and other multi disk features in Cockpit.

### CODE REVIEW

File: vm-run [hunk 1]
```diff
@@ -33,7 +33,8 @@ parser.add_argument('-G', '--graphics', action='store_true', help='Display a gra
 parser.add_argument('-q', '--quiet', action='store_true', help="Don't connect text console")
 parser.add_argument('-c', '--client', action='store_true', help="Open Cockpit Client (implies -q)")
 parser.add_argument('-C', '--cpus', default=None, type=int, help='Number of cpus in the target machine')
-parser.add_argument('-S', '--storage', default=None, type=str, help='Add a second qcow2 disk at this path')
+parser.add_argument('-S', '--storage', type=str, action='append',
+                    help='Add a qcow2 disk(s) at this path')
```
Comment (martinpitt) [inline vm-run:37]: Drop the `type=str` (it's the default anyway), and instead add `default=[]`. Then...

File: vm-run [hunk 2]
```diff
@@ -78,7 +79,8 @@ try:
     machine.start()
 
     if args.storage:
-        machine.add_disk(path=args.storage, type='qcow2')
+        for disk in args.storage:
```
Comment (martinpitt) [inline vm-run:82]: ... you can drop the `if` here, and just leave the `for`.

**Review by martinpitt — CHANGES_REQUESTED** [review]
Thanks! Let's clean this up a bit, but LGTM otherwise.

File: vm-run (hunk 1 above)
Comment (martinpitt) [reply vm-run:37]: While you are at it, `default=None` is pointless, you can also drop it from all other options. Perhaps do a first commit of dropping type=str and default=None?

Comment (jelly) [reply vm-run]: Good idea!

File: vm-run (hunk 2 above)
Comment (jelly) [reply vm-run]: That is a lot nicer, thanks!

**Review by martinpitt — APPROVED** [review]
Cheers!

**Pull request merged**